frontend/projects/ui/src/app/app.component.ts

```diff
@@ -133,19 +107,15 @@ export class AppComponent {
             ])
           })
         // UNVERIFIED
-      } else if (auth === AuthState.UNVERIFIED) {
```

Comment:
:question:  I'm looking for changes that prevent default. So the previous pattern, doesn't it make sure that all possible types are checked, if not there should be a way last time I used ts. Though this may only work with disjoint types rather than the hacked TS Enum
```ts
function assert_never(x: never) {}
let auth: AuthState = auth;
if (auth === AuthState.VERFIED) {
  // ..
} else {
  assertNever(auth)
}
```

Reply:
There's really just two options there, that's why I switched to boolean. I believe if there's a third one introduced — we would need to refactor anyway.

Reply:
Thanks